ConsumerApi.Tests.Integration/StepDefinitions/DevicesStepDefinitions.cs

```diff
@@ -1,222 +1,104 @@
-using Backbone.ConsumerApi.Tests.Integration.API;
+﻿using Backbone.BuildingBlocks.SDK.Endpoints.Common.Types;
+using Backbone.ConsumerApi.Sdk;
+using Backbone.ConsumerApi.Sdk.Authentication;
+using Backbone.ConsumerApi.Sdk.Endpoints.Devices.Types.Responses;
 using Backbone.ConsumerApi.Tests.Integration.Configuration;
-using Backbone.ConsumerApi.Tests.Integration.Models;
-using Backbone.Crypto;
-using Backbone.Crypto.Abstractions;
+using Backbone.ConsumerApi.Tests.Integration.Extensions;
 using Microsoft.Extensions.Options;
-using Newtonsoft.Json;
-using static Backbone.ConsumerApi.Tests.Integration.Helpers.ThrowHelpers;
 
 namespace Backbone.ConsumerApi.Tests.Integration.StepDefinitions;
 
 [Binding]
 [Scope(Feature = "DELETE Device")]
-internal class DevicesStepDefinitions : BaseStepDefinitions
+internal class DevicesStepDefinitions
 {
-    private readonly KeyPair _keyPair1;
-    private readonly KeyPair _keyPair2;
+    private Client? _sdk;
```

Comment:
Hmm, since we're using null suppression everywhere anyway, we might as well initialize the `_sdk` with `null`. And I know that I said otherwise in Daniel S.'s PR. But now that I see it heavily used, I don't think that it's a good idea to make it nullable.

Same for the other step definition files